pf/internal/convert/string.go

```diff
@@ -40,6 +40,12 @@ func (*stringEncoder) fromPropertyValue(p resource.PropertyValue) (tftypes.Value
 	if p.IsNull() {
 		return tftypes.NewValue(tftypes.String, nil), nil
 	}
+
+	// Special-case to tolerate booleans.
```

Comment:
This seems to be necessary now to accommodate AWS skip_metadata_api_check and pass our tests cases.